services/dy-static-file-server/dynamic-sidecar-compose-specs.yml

```diff
@@ -2,7 +2,11 @@ version: "3.7"
 services:
   dy-static-file-server-dynamic-sidecar-compose-spec:
     init: true
-    image: ${DOCKER_REGISTRY}/${DOCKER_IMAGE_NAME}-dynamic-sidecar-compose-spec:${DOCKER_IMAGE_TAG}
+    # - SIMCORE_REGISTRY will be replaced by the url of the simcore docker registry 
+    # deployed inside the platform
+    # - SERVICE_VERSION will be replaced by the version of the service
+    # to which this compos spec is attached
+    image: ${SIMCORE_REGISTRY}/${DOCKER_IMAGE_NAME}-dynamic-sidecar-compose-spec:${SERVICE_VERSION}
```

Comment:
DOCKER_IMAGE_NAME gets replaced when building the service right?

Reply:
yes. that one gets replaced during the build